vita3k/modules/SceNpBasic/SceNpBasic.cpp

```diff
@@ -41,8 +45,13 @@ EXPORT(int, sceNpBasicGetFriendListEntries) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceNpBasicGetFriendListEntryCount) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceNpBasicGetFriendListEntryCount, int *nb_friends) {
+    if (!nb_friends)
+        return RET_ERROR(SCE_NP_BASIC_ERROR_INVALID_ARGUMENT);
+
+    STUBBED("No friends");
+    *nb_friends = 0;
```

Comment:
This can be changed to 1 because you've always got a friend when I'm around :D

Reply:
or 2

Reply:
or 3